### ISSUE TITLE
Index-less folders are un-indented

### ISSUE THREAD

**Issue by Alzarath** [trendschau/typemill#429]
**Problem:** When a folder does not have an `index.md` or `index.txt` file, it fails to indent properly with the folder and the border color is white. Otherwise it functions normally.

Example in the file system:

```
starcraft-2/
├─ editor/
│  ├─ data/
│  │  ├─ channeled-spells
│  │  ├─ data-tokens
│  │  ├─ effects/
```

`data` has no index content file.

![indexless-filesystem](https://github.com/typemill/typemill/assets/1994142/58e5d47a-3b22-4a72-af4f-bc5eb5ad19dd)

**Workaround:** Include an `index.md` or `index.txt` file in the folder, empty or otherwise.

**Expectation:** I would expect the folder to behave as if it did have an empty content file.

It would also be nice if `index.yaml` applied to the folder without the content file. It currently does not.

**Comment by Alzarath** [comment]
I noticed the data folder isn't in the Frontend navigation, either. It occurs to me that the system uses .txt to store "pending" changes and .md to store "published" changes.

If I'm not misunderstanding, I think it would be best for the behavior to reflect as if there was an empty `index.txt` in the folder, with publishing resulting in the creation of an empty `index.md` file.

**Comment by trendschau** [comment]
> It occurs to me that the system uses .txt to store "pending" changes and .md to store "published" changes.

Yes exactly. There should be no folders without an index.txt file, they are created automatically when you create a folder in the interactive navigation. 

From your descriptions, I guess you have some special workaround or you add content manually?

**Comment by Alzarath** [comment]
Yes, I add content manually. I do not rely exclusively on TypeMill to view and modify my notes.

**Comment by trendschau** [comment]
I will close this since it is not an error and no task related.
